### PR TITLE
Add Shorthand for Install Command

### PR DESCRIPTION
**Pull request by hadihammurabi**
use username/repository for github
use ./example.AppImage for local file with relative path
use /dir/example.AppImage for local file with absolute path

example:
image

### CODE REVIEW

File: appimage.go [hunk 1]
```diff
@@ -53,6 +55,45 @@ func installAppImageOptionsFromCLIContext(context *cli.Context) (types.InstallOp
...
+		context.Set("github", "true")
+		context.Set("from", appName)
+		return context, splitted[1]
+	}
+
+	// local file with relative path
+	if len(splitted) == 2 && splitted[0] == "." {
+		pwd, err := os.Getwd()
+		if err != nil {
+			panic(err)
```
Comment (srevinsaju) [inline appimage.go:85]: But what about, when the provided command is?
```bash
zap install ./somedir/firefox.AppImage
```

**Review by srevinsaju — CHANGES_REQUESTED** [review]

**Comment by srevinsaju** [comment]
Looks really neat! Thank you for your patch! :tada:

**Comment by hadihammurabi** [comment]
I have fix thinks to change.
Please check

File: appimage.go [hunk 2]
```diff
@@ -12,7 +14,7 @@ import (
 
 func installAppImageOptionsFromCLIContext(context *cli.Context) (types.InstallOptions, error) {
 	executable := context.String("executable")
-	appName := context.Args().First()
+	context, appName := parseSourceFromAppName(context, context.Args().First())
```
Comment (srevinsaju) [inline appimage.go:17]: Is it possible to calculate the `appName` from the parameter too, like
```bash
zap install repo/software1
```
should automatically set the `appName = software1`

and for local files
```bash
zap install ./super-cool-x86_64.AppImage
```
it should set `appName = super-cool`

**Comment by srevinsaju** [comment]
@hadihammurabi are you still working on this, or shall I hand it over to @kadern0 ?

**Comment by hadihammurabi** [comment]
Yes, i still collecting cpu architectures types.

**Comment by srevinsaju** [comment]
@hadihammurabi it's there in the zap source code, https://github.com/srevinsaju/zap/blob/main/internal/helpers/arch.go, gives a list of supported architectures, but my proposed solution was more simpler than needing the list of architectures, but if it helps 🙂

**Pull request closed without merging**